developer-docs-site/docs/sdks/ts-sdk/typescript-sdk-overview.md

```diff
@@ -63,18 +63,27 @@ The OpenAPI client is a set of classes that are generated based on the Aptos RES
 
 ### Aptos Account
 
-The [class AptosAccount](https://aptos-labs.github.io/ts-sdk-doc/classes/AptosAccount.html) provides the methods for:
+The [class AptosAccount](https://aptos-labs.github.io/ts-sdk-doc/classes/AptosAccount.html) has a constructor that creates a new account instance or retrives an existing account instance, in addition it provides the methods for:
```

Comment:
Also to keep sentences short => "an existing account instance. In addition, ..."

Reply:
Done!